tests/testthat/test_display_functions.r

```diff
@@ -1,4 +1,4 @@
-context('display functions')
+context('general display functions')
 library(repr)
 
 last_data <- NULL
```

Comment:
i’d be happy to make the `test_display_func`/`get_last_data` dance DRY, but idk why.

only a refclass comes to mind:

```r
display_test <- DisplayTest()
[...]
withr::with_options(list(jupyter.base_display_func = test_display_func), {
[...]
expect_equal(display_test$get_last_data(), list(exp, NULL))
```